File: Networkcall/app/src/main/java/com/example/networkcall/CustomViewHolder.java
package com.example.networkcall;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class CustomViewHolder  extends RecyclerView.ViewHolder{
    ImageView ivItemBookCover;
    TextView tvItemBookTitle;


    public CustomViewHolder(@NonNull View itemView){
        super(itemView);

                ivItemBookCover = itemView.findViewById(R.id.iv_book_cover);
                tvItemBookTitle = itemView.findViewById(R.id.tv_book_title);
    }
}

File: RxJavaTest/app/src/main/java/com/example/rxjavatest/view/CustomViewHolder.kt
package com.example.rxjavatest.view

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.example.rxjavatest.R


class CustomViewHolder(itemView: View) :
    RecyclerView.ViewHolder(itemView) {
    var posterMovie : ImageView
    var movieTitle : TextView
    var movieGenre : TextView
    init {
        posterMovie = itemView.findViewById(R.id.iv_item_movies)
        movieTitle = itemView.findViewById(R.id.tv_title)
        movieGenre= itemView.findViewById(R.id.tv_genre)
    }
}